decidim-census/app/views/layouts/census/application.html.erb

```diff
@@ -2,8 +2,8 @@
 <html>
 <head>
   <title>Census</title>
-  <%= stylesheet_link_tag    "census/application", media: "all" %>
-  <%= javascript_include_tag "census/application" %>
+  <%= stylesheet_pack_tag "census/application", media: "all" %>
+  <%= javascript_pack_tag "census/application" %>
```

Comment:
these files not exists in the module. In fact, packs/ not exists.

Reply:
This layout is not being used anywhere. Controllers in this module make use of Decidim's admin layout.
Do you think there's any need to change this layout?

Reply:
Ok, let's just leave it then